### DISCUSSION TITLE
Missing Extensions?

### DISCUSSION THREAD

**Discussion by foqsi** [coder/code-server#5227] [Q&A] [answered]
Hey all,

I just installed code-server last night and I was looking at the extensions. I noticed I wasn't able to install C/C++ as I was on my local version of VSCode. I was wondering if I broke something, or if it just isn't possible.
![cpp](https://user-images.githubusercontent.com/81455653/170609218-4eb7a202-65c3-46c7-b1d7-6def9d5db699.png)
![cppp](https://user-images.githubusercontent.com/81455653/170609534-8e1a3aa1-ef9f-4650-b557-80a277aaabb7.PNG)

**Comment by jsjoeio** [comment, marked as answer]
Those are all Microsoft owned extensions. Chances are they aren't published on [OpenVSX](https://open-vsx.org/). That would be why they don't show up.

**Reply by foqsi** [reply]
> Those are all Microsoft owned extensions. Chances are they aren't published on [OpenVSX](https://open-vsx.org/). That would be why they don't show up.

Thank you for the information.
